src/Zicht/Bundle/MessagesBundle/Manager/MessageManager.php

```diff
@@ -279,20 +280,20 @@ public function check(TranslatorInterface $translator, $kernelRoot, $fix = false
                 $translator->setLocale($translation->locale);
                 $translated = $translator->trans($message->message, array(), $message->domain);
 
-                $translationFile = 'Resources/translations/' . $message->domain . '.' . $translation->locale . '.db';
-                if (!is_file($kernelRoot . '/' . $translationFile)) {
+                $translationFile = $kernelRoot . '/src/Resources/translations/' . $message->domain . '.' . $translation->locale . '.db';
```

Comment:
Dit zou alleen waar zijn in het geval van een nieuw project. Bij Sf4 upgrades adviseer ik de `src/Zicht/Bundle/WebsiteBundle/` file structuur te behouden.

Misschien is het beter om een slimmere detectie in te bouwen (zoeken naar de file location van de Kernel class bijv. of ...)

Reply:
Dit gaat over het opzoeken van onze `.db` file die in `Resources/translations` staat in de root van het project.
Aangezien we support droppen voor < SF4 kunnen we in dit geval dezelfde werking aanhouden en de `.db` bestanden in `/translations` zetten en zou de `translationFile` (misschien renamen naar `translationDBFile` ?) zich in `$kernelRoot . '/translations'` bevinden

Reply:
Aha ok, dat maakt het een stuk eenvoudiger. Want bij project die geupgrade worden staan die `.db` files ook gewoon in `/translations/` en niet in `src/Resources/translations/` :smile:

Reply:
Alleen weet ik niet of dit nog klopt:
>  * Using translations is/seems to be dependent on the order of scanned directories.
  By using this directory we assure we are the last in line, overwriting templates with database.

Reply:
Voor zover ik kan zien is dit wat er aan de hand is. En dat is een eenvoudige aanpassing. Maar dubbelcheck die code in die MessageManager nog ff goed :grimacing: